locales/en.json

```diff
@@ -78,17 +78,17 @@
   "export.map_view_expl": "bbox, center, etc.",
 
   "export.section.data": "Data",
-  "export.as_geoJSON": "as <a id=\"export-geoJSON\" href=\"\">geoJSON</a>",
-  "export.as_GPX": "as <a id=\"export-GPX\" href=\"\">GPX</a>",
-  "export.as_KML": "as <a id=\"export-KML\" href=\"\">KML</a>",
+  "export.as_geoJSON": "<a id=\"export-geoJSON\" href=\"\">download</a>/<a id=\"copy-geoJSON\" href=\"\">copy</a> as GeoJSON",
```

Comment:
maybe add `title="copies export output to clipboard"` (or similar) attributes to get a tooltip which makes it clear what *copy* means in this context? … and if we're already at that: the "download" option could have such a tooltip reading "saves the exported data as a file".